src/vaults/SectorVault.sol

```diff
@@ -224,12 +224,12 @@ contract SectorVault is SectorBase {
 		/// value here is the fraction of the shares owned by the vault
 		/// since the xVault doesn't know how many shares it holds
 		uint256 xVaultShares = balanceOf(_msg.sender);
-		uint256 shares = (_msg.value * xVaultShares) / 1e18;
+		uint256 shares = (_msg.value * xVaultShares) / 100;
```

Comment:
percentages / fractions should always be in the form of `100% = 1e18`

Reply:
done